variables.tf

```diff
@@ -259,12 +259,18 @@ variable "tags" {
   default     = {}
 }
 
-variable "vault_pillar_extra_config" {
-  type = list(object({
-    name   = string
-    type   = string # this can be auth, secrets, policies, or audit
-    config = map(string)
-  }))
+variable "pillar_template_vars" {
   description = "(Optional) List extra configurations to be referenced in the pillar"
+  default     = {}
+}
+
+variable "pillar_ssm_parameters_prefix" {
```

Comment:
I feel like there should be a way to avoid needing this variable. Just use `pillar_ssm_parameter_keys` to generate the resource arns for the iam policy

Reply:
Don't think we need this anymore since CB will be retrieving the value from SSM parameter.